calico-enterprise/getting-started/upgrading/upgrading-enterprise/kubernetes-upgrade-tsee/helm.mdx

```diff
@@ -66,39 +66,40 @@ These steps differ based on your cluster type. If you are unsure of your cluster
        : 'curl -O -L {{downloadsurl}}/ee/charts/tigera-operator-{{chart_version_name}}.tgz'}
    </CodeBlock>
 
-1. Install the {{prodname}} custom resource definitions.
+2. Install the {{prodname}} custom resource definitions.
 
    ```bash
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/operator-crds.yaml
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/prometheus-operator-crds.yaml
    kubectl apply --server-side --force-conflicts -f {{filesUrl}}/manifests/eck-operator-crds.yaml
    ```
 
-1. Run the Helm upgrade command for `tigera-operator`:
+3. Run the Helm upgrade command for `tigera-operator` and make sure to either update `values.yaml` with your configuration or use custom `values.yaml` file:
```

Comment:
```suggestion
1. Run the Helm upgrade command for `tigera-operator` and make sure to either update `values.yaml` with your configuration or use custom `values.yaml` file:
```

Reply:
why are we changing this from 3. to 1. ?

Reply:
got your point! but had to move :::note::: to the end of the page, otherwise the ordered list numbering was not working with having both multiline <CodeBlock> and :::note::: under #3.